docs/content/en/install/local/multi-cluster-helm/_index.md

```diff
@@ -16,9 +16,18 @@ If you haven't already, see the install [prerequisites]({{< relref "install/loca
 
 Deploy K8ssandra with multiple Cassandra datacenters in a **multi-cluster** kind environment.
 
-### Clone the repo and use the setup script
+### Add the K8ssandra Helm chart repo
```

Comment:
Only thing I can think of: if we add this section to the prerequisites, it could perhaps be duplicative if it's also on the install pages themselves. Of course, that would assume people actually follow the documentation 😉

Reply:
Yeah it's duplicate reminders, but my thought was to cover all the bases (in prereqs topic, and in the install with helm topics) to ensure users notice it whether they read docs sequentially or if they land on this one directly via leftnav or search.